.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release new version
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        description: Type of release based om semver (e.g. patch, minor, major)
+        required: true
+        default: minor
+
+jobs:
+  release:
+    name: Release new version to NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Setup Git
+        run: |
+          git config user.name github-actions[bot]
```

Comment:
(still approved but curious)
Is this a real GitHub user or are we just setting the git name to this (semi) arbitrarily?

Reply:
It is a real GitHub user. I forgot where I originally got it from, but there's some validation here: https://github.community/t/github-actions-bot-email-address/17204/5?u=neverendingqs

Example commit created by this user: https://github.com/neverendingqs/serverless-default-aws-resource-properties/commit/aca362d77790734f9c12669c91025143fba17bda